### DISCUSSION TITLE
Windows 11

### DISCUSSION THREAD

**Discussion by bxlogic** [Ylianst/MeshCentral#3832] [General]
Dear Ylinast,

I want to thank you for this beautiful opportunity to test and see the miracle you made, i have installed it in 100 computers in windows 11 and everything works like charm i plan also to install it on 200 more, i had no problems except i figure it out for the agent config file i had latency, and instead of name: local i added it the ip address and the path to the agent: wss://mylanip:443/agent.ashx and i have no more latency issue, I'd suggest to work more on the android app to connect also in LAN not just WAN because i work in a university, and we use android smartboards, so it would be perfect if there's such variant to install it.

I'm into the IT sector 25 years, i also could contribute on testing and UI customization, make you a great design, logo, and icons it all deppends what do you wish and have a need to, I would be happy to help you maintain and grow this wonderful project.

My best regards.

**Comment by Ylianst** [comment]
Oh thank you. For Android in LAN mode, it's possible I could add it. The Android agent would multicast in the local network to find the server and then connect to it. I am currently plenty busy with lots of requests, but at some point I will make another run at the Android agent and see if I can add LAN mode support, improve display remote control and access more files. I am just completely swamped currently. If you can contribute anything, that would be wonderful.
